src/universal/conf/private-testnet.conf

```diff
@@ -12,12 +12,21 @@ mantis {
     # Set to false to disable broadcasting the NewBlockHashes message, as its usefulness is debatable,
     # especially in the context of private networks
     broadcast-new-block-hashes = false
+
+    pruning {
+        mode = "archive"
+    }
   }
 
   blockchains {
     network = "private"
   }
 
+  consensus {
+    coinbase = "0011223344556677889900112233445566778899" # has to be changed for each node
```

Comment:
why this is part of the default?

Reply:
I wanted to have anything here(it is the default from application.conf). For each node, we have to obtain a different account